t/is-and-extract-ipv6-gua.pl

```diff
@@ -0,0 +1,472 @@
+use Test::More;
+use B qw(perlstring);
+
+SKIP: { eval { require Test::Warnings; } or skip($@, 1); }
+eval { require 'ddclient'; } or BAIL_OUT($@);
+
+my @valid_ipv6_not_global = (
+    # Unassigned address
+    "::",
+    "::0",
+    "::0:0",
+    "::0:0:0:0:0:0:0",
+    "0:0:0:0:0:0:0:0",
+    "0::0",
+    # link local address
+    "::1",
+    "::0:0:0:01",
+    "::0:0:0:001",
+    "::0:0:0:0001",
+    "0::1",
+    "0:0::1",
+    "0:0:0:0:0:0:0:1",
+    # Must start with a non-zero xxxx::/16
+    "::123.123.123.123",
+    "::2222:3333:4444:5555:6666:123.123.123.123",
+    "::3333:4444:5555:6666:123.123.123.123",
+    "::4444:5555:6666:123.123.123.123",
+    "::5555:6666:123.123.123.123",
+    "::6666:123.123.123.123",
+    "::ffff:192.168.1.1",
+    # Unique local address (ULA)
+    "FC11:2222:3333:4444:5555:6666:7777:8888",
+    "FC11::6666:7777:8888",
+    "FC11::",
+    "FC11:2222::123.123.123.123",
+    "FD11:2222:3333:4444:5555:6666::8888",
+    "FD11:2222:3333:4444:5555::8888",
+    "FD11:2222::8888",
+    "FD11::3333:4444:5555:6666:123.123.123.123",
+    "FD11::4444:5555:6666:123.123.123.123",
+    "FD11::5555:6666:123.123.123.123",
+    "FD11::6666:123.123.123.123",
+    "FD11::8888",
```

Comment:
Just include the first and last address of the range:

```suggestion
    "fc00::",
    "fdff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
```

Reply:
Done